.github/workflows/test.yml

```diff
@@ -7,7 +7,7 @@ jobs:
   tests:
     strategy:
       matrix:
-        os: [ ubuntu-latest, macos-12 ]
+        os: [ ubuntu-latest, macos-12, macos-13 ]
```

Comment:
`macos-12` will be removed in a follow-up PR, as removing it now renders the CI stuck. Maybe there is a better way, I just don't know...

Reply:
Okay, this works now, IDX changed the required check.